package.json

```
@@ -9,13 +9,13 @@
         "prettier:fix": "pnpm prettier . --write",
         "eslint:check": "pnpm eslint .",
         "eslint:fix": "pnpm eslint --fix .",
-        "build:dev": "rollup -cw",
+        "build:dev": "rollup -c --watch",
```

```suggestion
        "build:dev": "rollup --config --watch",
```
If we are already getting more explicit, let's also make that for the other argument